### PR TITLE
Add missed column title in runner management page

### PR DESCRIPTION
**Pull request by yp05327**
image

### CODE REVIEW

**Review by silverwind — CHANGES_REQUESTED** [review]
Don't add a new translation for something that was already translated (multiple times!) and instead use `edit` from top level (options/locale/locale_en-US.ini line 88).

File: templates/shared/actions/runner_list.tmpl [hunk 1]
```diff
@@ -52,6 +52,7 @@
 					<th>{{.locale.Tr "actions.runners.owner_type"}}</th>
 					<th>{{.locale.Tr "actions.runners.labels"}}</th>
 					<th>{{.locale.Tr "actions.runners.last_online"}}</th>
+					<th>{{.locale.Tr "actions.runners.edit"}}</th>
 					<th></th>
```
Comment (wxiaoguang) [inline templates/shared/actions/runner_list.tmpl]: There is an empty `<th>` below, this column is left as empty by design, I do not think it needs a "column title"

**Comment by yp05327** [comment]
@silverwind 
The reason why I added `runners.edit` is because I saw some same `edit` in different places, so maybe they need to use the top level `edit` too?
image
image

File: templates/shared/actions/runner_list.tmpl (hunk 1 above)
Comment (yp05327) [reply templates/shared/actions/runner_list.tmpl]: Ah, fogot to remove it.

ps: The root reason of this PR is that I found that there's a word break of `Last Online Time` which seems strange.
image

Then I found that in other pages, `edit` column has a title.
image
image

**Review by wxiaoguang — APPROVED** [review]

**Review by wolfogre — APPROVED** [review]

**Comment by silverwind** [comment]
Yes, they should all share ideally. There are probably numerous translations that have the same issue. In this case it is a easy fix because the top-level key exists.

I would like us to try moving other duplication translations to top-level where it makes sense. Previously it was said we should not do because it may break crowdin, but I think it's worth a try. Maybe make a script where one could input a translation key and have it moved to top-level in all language files and see how that goes.

**Review by silverwind — APPROVED** [review]

**Comment by yp05327** [comment]
I see the reason why no `Edit` here.
If the user have no permission to edit the runner, it will be empty which looks strange. 
I will improve this in another PR.
image

**Comment by wxiaoguang** [comment]
So if a column doesn't provide necessary information , then it is optional 😁

**Pull request merged**